java/rocksjni/write_batch_with_index.cc

```diff
@@ -860,3 +860,15 @@ jlongArray Java_org_rocksdb_WBWIRocksIterator_entry1(JNIEnv* env,
 
   return jresults;
 }
+
+/*
+ * Class:     org_rocksdb_WBWIRocksIterator
+ * Method:    refresh0
+ * Signature: (J)V
+ */
+void Java_org_rocksdb_WBWIRocksIterator_refresh0(JNIEnv* env, jobject /*jobj*/,
+                                            jlong handle) {
+  ROCKSDB_NAMESPACE::Status s = ROCKSDB_NAMESPACE::Status::NotSupported("Refresh() is not supported");
+  (void)handle;
```

Comment:
What is the `(void)handle;` line for?

Reply:
Apart from that it looks good :-)

Reply:
Suppress unused parameter warning. Could have commented out the name, but eh

Reply:
My bad, I wasn't aware you could comment out the name instead :sweat_smile:

Reply:
@npepinpe IMHO it's better to remove the names that are unused rather than commented out - could you update in that manner please? I think @pdillinger has the facility to re-import the PR?